packages/experience/src/App.tsx

```diff
@@ -39,14 +41,21 @@ import Springboard from './pages/Springboard';
 import VerificationCode from './pages/VerificationCode';
 import { UserMfaFlow } from './types';
 import { handleSearchParametersData } from './utils/search-parameters';
-
 import './scss/normalized.scss';
 
 handleSearchParametersData();
 
 const App = () => {
   return (
     <BrowserRouter>
+      <Helmet
+        htmlAttributes={{
+          // We intentionally use the imported i18next instance instead of the hook, since the hook
+          // will cause a re-render following some bugs here. This still works for the initial
+          // render, so we're good for now. Consider refactoring this in the future.
+          lang: i18next.language,
+        }}
+      />
```

Comment:
Seems redundant now as we found it already set in `AppMeta.tsx`.

https://github.com/logto-io/logto/blob/master/packages/experience-legacy/src/Providers/AppBoundary/AppMeta.tsx#L41

Reply:
updated